### ISSUE TITLE
Share directly by Android App 

### ISSUE THREAD

**Issue by Coyoter** [JakeLane/Toffeed#19]
Hopefully Toffeed have function to share content directly by Android. For example, share via chrome.

( I means, the native Android share function , now we can't share content directly to Toffeed)

**Comment by Coyoter** [comment]
It's different of #8

When I surfing on internet by Chrome, want share link to Facebook, can't find Toffeed at Chrome (or anything else) native share function.

**Comment by twestheimer** [comment]
Toffeed is amazing BUT I really miss not being able to share from photos and chrome!
Thanks

**Comment by JakeLane** [comment]
I'm gonna try and get this done this weekend, stay tuned!

**Comment by twestheimer** [comment]
I'll second the "that would be amazing" comment

**Comment by JakeLane** [comment]
I've had a go at implementing this but ran into a roadblock. Apparently, you can't change the upload input via JavaScript (which makes a lot of sense for security reasons). I'm not really sure how to work around this so I'm gonna leave this for a bit.

Sorry about that.

**Comment by twestheimer** [comment]
how do people share pictures and websites presently?
thanks

On Sat, Mar 19, 2016 at 7:44 AM, Jake Lane notifications@github.com wrote:

> I've had a go at implementing this but ran into a roadblock. Apparently,
> you can't change the upload input via JavaScript (which makes a lot of
> sense for security reasons). I'm not really sure how to work around this so
> I'm gonna leave this for a bit.
> 
> Sorry about that.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> https://github.com/JakeLane/Toffeed/issues/19#issuecomment-198688503

## 

---

_To a Buddhist "Self Control" is either a double negative and/or an
oxymoron Be loyal to peace, justice, the social good Patriotism is your
conviction that this country is superior to all other countries because you
were born in it.      George Bernard Shaw"Your assumptions are your windows
to the world.  Scrub them off every once in a while, or light won't come
in."_
Tom Westheimer  _US  phone 603-525-7733 (calls cell and home) *
ทอม  เวสต์ไฮม์เมอร์  Thai phone
*+6685-4466-954_

**Comment by JakeLane** [comment]
You'll be able to share URLs from chrome, but you won't be able to share images to Toffeed directly.

**Comment by Coyoter** [comment]
Jake,  you can check an app named 'metal for Facebook & Twitter' , it's a
browser fb app too, but it let you share links from chrome and upload
photos from your gallery.

fyi

Jake Lane notifications@github.com 於 2016年3月19日週六 20:33 寫道：

**Comment by creativetrendsapps** [comment]
Yes @Coyoter is correct. I doubt the developer will help with the code though @JakeLane

**Comment by twestheimer** [comment]
could one use
IFTTT https://ifttt.com/
​ ?​

On Sat, Mar 19, 2016 at 8:33 AM, Jake Lane notifications@github.com wrote:

> You'll be able to share URLs from chrome, but you won't be able to share
> images to Toffeed directly.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> https://github.com/JakeLane/Toffeed/issues/19#issuecomment-198694935

## 

---

_To a Buddhist "Self Control" is either a double negative and/or an
oxymoron Be loyal to peace, justice, the social good Patriotism is your
conviction that this country is superior to all other countries because you
were born in it.      George Bernard Shaw"Your assumptions are your windows
to the world.  Scrub them off every once in a while, or light won't come
in."_
Tom Westheimer  _US  phone 603-525-7733 (calls cell and home) *
ทอม  เวสต์ไฮม์เมอร์  Thai phone
*+6685-4466-954_

**Comment by JakeLane** [comment]
I think it would be possible to intercept the HTTP response and edit it before presenting to the user. That would take a bit of research, so maybe next release.
